pkg/plugin/v3/scaffolds/internal/templates/config/api/hub_conversion.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/pkg/model/file"
+)
+
+var _ file.Template = &HubConversion{}
+
+type HubConversion struct { // nolint:maligned
+	file.TemplateMixin
+	file.MultiGroupMixin
+	file.BoilerplateMixin
+	file.ResourceMixin
+
+	HubVersion string
+}
+
+func (c *HubConversion) SetTemplateDefaults() error {
+	c.Resource.Version = c.HubVersion
+	if c.Path == "" {
+		if c.MultiGroup {
+			c.Path = filepath.Join("apis", "%[group]", "%[version]", "%[kind]_conversion.go")
+		} else {
+			c.Path = filepath.Join("api", "%[version]", "%[kind]_conversion.go")
+		}
+	}
+	c.Path = c.Resource.Replacer().Replace(c.Path)
+	fmt.Printf("Scaffolding conversion logic for hub version %s in path %s", c.Resource.Version, c.Path)
```

Comment:
IHMO we should not add prints here. Have we any other boilerplate printing. the actions?
If we would like to log out the action then, I think it needs to be in the plugin which is reasonable for the execution. 
The boilerplate just define the template which will be executed. so, I do not think that it is part of its domain of responsibility.

Reply:
Had added this, because webhook conversion scaffolding prints the path: [here](https://github.com/kubernetes-sigs/kubebuilder/blob/49f7a5eb70f85034473ff18c85827337f3c7d9a9/pkg/plugin/v3/scaffolds/internal/templates/config/api/webhook.go#L59). Shall I remove it or still have it ? Im fine with removing it too.